sdk/lib/crt/misc/dbgrpt.cpp

```diff
@@ -377,11 +373,7 @@ int __cdecl _CrtDbgReportW(int reportType, const wchar_t *filename, int linenumb
 
     if (format)
     {
-        va_list arglist;
-        va_start(arglist, format);
-        int len = _vsnwprintf(szFormatted, DBGRPT_MAX_BUFFER_SIZE - 2 - sizeof(DBGRPT_ASSERT_PREFIX_MESSAGE), format, arglist);
-        va_end(arglist);
-
+        int len = _vsnwprintf(szFormatted, DBGRPT_MAX_BUFFER_SIZE - 2 - sizeof(DBGRPT_ASSERT_PREFIX_MESSAGE), format, argptr);
```

Comment:
I'm not sure about this.

```suggestion
        int len = _vsnwprintf(szFormatted, DBGRPT_MAX_BUFFER_SIZE - 2 - _countof(DBGRPT_ASSERT_PREFIX_MESSAGE), format, argptr);
```